src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -15,7 +18,10 @@ namespace System.Runtime.InteropServices
 
         public static OSPlatform Linux { get; } = new OSPlatform("LINUX");
 
-        public static OSPlatform OSX { get; } = new OSPlatform("OSX");
+        public static OSPlatform macOS { get; } = new OSPlatform("MACOS");
+
+        [EditorBrowsable(EditorBrowsableState.Never)] // https://github.com/dotnet/runtime/issues/33331#issuecomment-650326500
```

Comment:
Is this link really useful for context? It points to several screens of API review notes, and it does not say much useful about the OSX.

Should this rather say "// superseded by macOS" ?